packages/css-framework/src/abstracts/functions/_index.scss

```diff
@@ -0,0 +1,6 @@
+@forward "color";
+@forward "map-deep-get";
+@forward "non-destructive-map-merge";
+@forward "spacing";
+@forward "typography";
+@forward "map-get-default";
```

Comment:
Can we have a newline saved to the end of this file? There are a few others like this I think.

There should be a plugin for your IDE that respects the .editorconfig (https://editorconfig.org/) file at the repo root.

Reply:
Only installed the plugin halfway through this PR, so I've probably missed a few spots. Thanks though!